### PR TITLE
Migration notice

### PR DESCRIPTION
**Pull request by xuzhang3**
## All Submissions:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] I have updated the documentation accordingly.
* [x] I have added tests to cover my changes.
* [x] All new and existing tests passed.
* [x] My code follows the code style of this project.
* [x] I ran lint checks locally prior to submission.
* [x] Have you checked to ensure there aren't other open PRs for the same update/change?

## What about the current behavior has changed?

Issue Number:

## Does this introduce a change to `go.mod`, `go.sum` or `vendor/`?

- [ ] Yes
- [x] No

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Any relevant logs, error output, etc?

(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
+
+For more on this, please see the below posts  
+[Azure DevOps Provider for Terraform](https://cloudblogs.microsoft.com/opensource/2020/06/18/announcing-hashicorp-terraform-azure-devops-provider-release)    
+[Azure DevOps provider document](https://www.terraform.io/docs/providers/azuredevops/index.html)
+
 # Terraform Provider for Azure DevOps (Devops Resource Manager)
 
 [![Gitter](https://badges.gitter.im/terraform-provider-azuredevops/community.svg)](https://gitter.im/terraform-provider-azuredevops/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
-[![Go Report Card](https://goreportcard.com/badge/github.com/microsoft/terraform-provider-azuredevops)](https://goreportcard.com/report/github.com/microsoft/terraform-provider-azuredevops)
+[![Go Report Card](https://goreportcard.com/badge/github.com/terraform-providers/terraform-provider-azuredevops)](https://goreportcard.com/report/github.com/terraform-providers/terraform-provider-azuredevops)
```
Comment (EliiseS) [inline README.md:11]: Instead on making changes to the docs, should we just remove all content here and link to the new documentation in the new repo? 

Also, should we be changing the import statements to the new repo? I think it makes more sense to just keep it this repo and not make any changes to the code.

Comment (xuzhang3) [reply README.md:11]: My mistake. I import all the changes from new repo.  What I want is update the `README.ME`,  follow the introductions at https://github.com/microsoft/vscode-go.

**Comment by xuzhang3** [comment]
Closing this issue for unwanted changes.

File: README.md (hunk 1 above)
Comment (tmeckel) [reply README.md:11]: @xuzhang3 I agree with @EliiseS here. This repo should be archived and set to readonly. 

image

**Pull request closed without merging**